connector/src/main/java/com/google/cloud/hive/bigquery/connector/input/BigQueryInputSplit.java

```diff
@@ -44,9 +47,6 @@
 import org.apache.hadoop.mapred.JobConf;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
-import shaded.hivebqcon.com.google.cloud.bigquery.connector.common.*;
-import shaded.hivebqcon.com.google.common.annotations.VisibleForTesting;
-import shaded.hivebqcon.com.google.common.collect.ImmutableList;
```

Comment:
don't know if the issue of java.lang.NoSuchFieldError: VIRTUAL_COLUMN_NAMES is relate to the change here.